netty/src/main/java/io/grpc/netty/NettyServerHandler.java

```diff
@@ -289,7 +290,8 @@ public void handlerAdded(final ChannelHandlerContext ctx) throws Exception {
 
     // init max connection age monitor
     if (maxConnectionAgeInNanos != MAX_CONNECTION_AGE_NANOS_DISABLED) {
-      maxConnectionAgeMonitor = ctx.executor().schedule(
```

Comment:
Whe mocking, `ctx.executor()` returns the correct type,: `io.netty.channel.EventLoop$$EnhancerByMockitoWithCGLIB$$d741cc0d.schedule(<generated>`. But when it calls `.schedule()`, it will then invoke the delegate:

`method.invoke(fakeClock, arguments)`, where `method` is `EventLoopGroup.schedule()`, `fakeClock` is instance of `FakeClock` not `EventLoopGroup`.